content/en/docs/release-notes/rn-armory-agent/agent-plugin/agent-plug-v-0-10-3.md

```diff
@@ -0,0 +1,14 @@
+---
+title: v0.10.3 Armory Agent Clouddriver Plugin (2021-10-22)
+toc_hide: true
+version: 00.10.03
+
+---
+
+## Changes
+
+* When migrating from the Kubernetes V2 provider to the Agent, the cache for the V2 provider gets evicted now. The eviction happens in batches. You can tune this behavior with the following parameters:
+
+  * `kubesvc.v2-cache-eviction.disable`: (Boolean) Set this to `true` if you want to turn off the eviction of the V2 cache.
+  * `kubesvc.v2-cache-eviction.batch-size`: (Integer) How many accounts to evict for each eviction event. (Default 5)
```

Comment:
**How many kubernetes kinds*** do not add this, only for context: (the eviction finishes until there are no more kinds to evict, each account would hold a variable amount, greater batch-size = lesser trips to the db but more overhead in clouddriver)